website/docs/releases/2024/v2024.2.md

```diff
@@ -0,0 +1,10180 @@
+
+-   **Session location and network binding**
+
+    Sessions for any users can now be bound to a specific geolocation (Continent, Country, City) or network (Autonomous System, subnet, IP address). If the session is accessed from a different location/network it was initially created from, the session will be terminated.
+
+    Configuration steps are available [here](../../flow/stages/user_login/index.md#network-bindinggeoip-binding).
+
+-   **S3 file storage**
+
+    Media files can now be stored on S3. Follow the [setup guide](../../installation/storage-s3.md) to get started.
+
+-   **_Pretend user exists_ option for Identification stage**
+
+    Previously the identification stage would only continue if a user matching the user identifier exists. While this was the intended functionality, this release adds an option to continue to the next stage even if no matching user was found. "Pretend" users cannot authenticate nor receive emails, and don't exist in the database.
+
+    **This feature is enabled by default.**
+
+-   **Tenancy** <span class="badge badge--primary">Enterprise</span>
+
+    :::warning
+    This feature is in early preview. Use at your own risk.
+    :::
+
+    It allows for authentik operators to manage several authentik installations without having to deploy additional instances.
+
+
+### UX features
+
+-   **Simplified workflow for creating applications and providers**
+
+    Applications and providers can now be created at the same time using the **Application Wizard**, found on the **Applications** -> **Applications** page of the admin interface, skipping the linking step between the two.
+
+-   **Ability to select more than 20 providers in an outpost**
+
+    We have introduced a new way of selecting providers in the outpost configuration form, which allows for more than 20 providers to be selected for a single outpost.
+
+    This dual-list multiselect prompt is only available for outposts for now, but we plan on extending it to other forms.
+
+-   **Attribute preview per user**
+
+    You can now preview attributes transmitted to SAML and OAuth applications for a specific user.
+
+-   **Display applications a user has access to**
+
+    An administrator can now see all applications a user has access to on the user's page in the admin interface.
+
+### Other noteworthy features
+
+-   **New graph for event volume**
+
+    In the **Events** -> **Log** page in the admin interface, you can see a graph of the event volume matching the search query over the last 7 days.
+
+-   **Flows can now be restricted to outposts**
+
+    You can now restrict a flow to be used only by an outpost using the **Require Outpost** setting on the flow. This would mainly be used to LDAP flows.
```

Comment:
```suggestion
    You can now restrict a flow to be used only by an outpost using the **Require Outpost** setting on the flow. This would mainly be used for LDAP flows.
```